cmd/activator/main.go

```diff
@@ -191,7 +191,7 @@ func main() {
 	configStore.WatchConfigs(configMapWatcher)
 
 	// Open a WebSocket connection to the autoscaler.
-	autoscalerEndpoint := fmt.Sprintf("ws://%s.%s.svc.%s%s", "autoscaler", system.Namespace(), pkgnet.GetClusterDomainName(), autoscalerPort)
+	autoscalerEndpoint := fmt.Sprintf("ws://%s.%s.svc.%s%s", "autoscaler-ss", system.Namespace(), pkgnet.GetClusterDomainName(), autoscalerPort)
```

Comment:
In K8s APIs, a statefulset is abbreviated as `sts`. Should we do that throughout? It'd have the additional benefit of not making me think about nazi germany when reading those pieces of code.

Reply:
done